Makefile

```diff
@@ -10,7 +10,9 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-OS := $(shell uname -s)
+ifneq ($(OS), Windows_NT)
+	OS := $(shell uname -s)
+endif
```

Comment:
I'm now a bit confused by this change. Assuming we're running Windows, then `shell uname -s` returns `Windows_NT` anyways, so wouldn't the value of `OS` be the same regardless of the ifneq?

Reply:
Ah, so in your case `OS` is pre-defined to be `Windows_NT` but running `shell uname -s` returns `CYGWIN_NT...`, I see. 

Follow-up question: if someone is running windows subsytem for linux (WSL), will the current setup also work? Or can we get in trouble because `OS` is pre-defined for `Windows_NT` but we need the `*.so` file for compilation, as we're on WSL?

Reply:
I installed WSL and `uname` returns `Linux` and `OS` is not predefined, so I think there is no problem

Reply:
Great, thanks for checking!